src/FrameManager.ts

```diff
@@ -68,7 +68,7 @@ class FrameManager {
 
     this._iframe.setAttribute(
       'sandbox',
-      'allow-scripts allow-same-origin allow-modals allow-forms'
+      'allow-scripts allow-same-origin allow-modals allow-forms  allow-popups'
```

Comment:
Super minor but looks like an extra space here

Reply:
might have been the merge conflict resolution that got me there.  thanks